docs/src/main/sphinx/release/release-465.md

```diff
@@ -0,0 +1,71 @@
+# Release 465 (15 Nov 2024)
+
+## General
+
+* Add the {func}`cosine_similarity` function for dense vectors. ({issue}`23964`)
+* Add support for reading geometries in [EWKB
+  format](https://en.wikipedia.org/wiki/Well-known_text_representation_of_geometry)
+  with the {func}`ST_GeomFromBinary` function. ({issue}`23824`)
+* Add support for the `ORDER BY` clause in a windowed aggregate function. ({issue}`23929`)
+* Improve performance of queries with selective joins on large build sides. ({issue}`22824`)
+* Fix potential configuration leakage between catalogs using JDBC-based
+  connectors. ({issue}`24058`)
+* Reduce chance of out of memory query failure when `retry-policy` is set to
+  `task`. ({issue}`24114`)
+
+## CLI
+
+* Fix incorrect quoting of output values when the `CSV_UNQUOTED` or the 
+  `CSV_HEADER_UNQUOTED` format is used. ({issue}`24113`)
+
+## BigQuery connector 
+
+* Fix failure when reading views with `timestamp` columns. ({issue}`24004`)
+
+## Cassandra connector
+
+* {{breaking}} Require setting `cassandra.security=PASSWORD` along with
+  `cassandra.username` and `cassandra.password` for  password-based
+  authentication. ({issue}`23899`)
+
+## Clickhouse connector
+
+* Fix insert for tables with the Clickhouse `TimestampWithTimeZone` type. ({issue}`23785`)
+
+## Delta Lake connector
+
+* Fix incorrect results when filtering partition columns on checkpoints with
+  `NAME` column mapping mode. ({issue}`24104`)
+
+## Faker connector
+
+* Add the `random_string` catalog function to test expressions for random string
+  creation. ({issue}`23990`)
+* Make generated data deterministic for repeated queries. ({issue}`24008`)
+
+## Hive connector
+
+* Add support for skipping archiving when committing to a table in the Glue metastore
+  and the `hive.metastore.glue.skip-archive` configuration property is set to
+  `true`. ({issue}`23817`)
+* Use drop and create operation when replacing an existing SQL routine stored in
+  the Hive Metastore. ({issue}`24010`)
+
+## Iceberg connector
+
+* Add support for reading and writing arbitrary table properties with the
+  `extra_properties` table property. ({issue}`17427`, {issue}`24031`)
+* Add support for configuring an OAuth2 server URI with the
+  `iceberg.rest-catalog.oauth2.server-uri` configuration property. ({issue}`23086`)
+* Improve performance of Iceberg queries involving multiple table scans. ({issue}`23945`)
+
+
+## Redshift connector
+
+* Add varchar/char cast pushdown support. ({issue}`23808`)
```

Comment:
Is this a for casts *from* or *to* varchar/char?

Reply:
not sure .. they PR talks about cast projection pushdown. @mayankvadariya ?

Reply:
cast from `varchar` to `varchar` and `char` to `char` having different precisions.

Reply:
Updated